python/desc/imsim/imSim.py

```diff
@@ -178,3 +202,32 @@ def extract_objects(df):
     phosim_galaxies = \
         phosim_galaxies.query("halfLightSemiMajor >= halfLightSemiMinor")
```

Comment:
"halfLightSemiMajor >= halfLightSemiMinor"

Was an error handling case.  There was only one of these in the 80000 item test catalog that Scott made.  So really, it should be fixed in the production program.

But, for here, it doesn't feel right to me to just put it into the query like this.  There might be other mistakes that we want to check for too.  So, if we want to include the selection here maybe it would be better to put these in a string called "validGalaxyCondition" or something.